src/status_im/navigation/screens.cljs

```diff
@@ -500,6 +501,12 @@
      :options   options/transparent-modal-screen-options
      :component saved-addresses-settings/view}
 
+    {:name      :screen/settings.keypairs-and-accounts
+     :options   (merge
+                 options/transparent-modal-screen-options
+                 options/dark-screen)
```

Comment:
nit: This seems a bit confusing since both maps have duplicated kvs, I'd just create a new map.